EIPS/eip-7512.md

```diff
@@ -18,71 +18,89 @@ The proposal aims to create a standard for an onchain representation of audit re
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
 ### Audit Properties
 
 - Auditor
-    - `name` - Name of the auditor (i.e. for displaying to the user)
-    - `uri` - Uri to retrieve more information about the auditor
-    - `authors` - A list of authors that contributed to this audit. This SHOULD be the persons that audited the contracts and created the audit.
+    - `name`: Name of the auditor (i.e. for displaying to the user)
+    - `uri`: URI to retrieve more information about the auditor
+    - `authors`: A list of authors that contributed to this audit. This SHOULD be the persons who audited the contracts and created the audit
 - Audit
-    - `auditor` - Information on the auditor
-    - `contract` - MUST be the `chainId` and `address` of the deployed contract the audit is related to.
-    - `issuedAt` - MUST contain the information when the original audit (identified by the `auditHash`) was issued
-    - `ercs` - A list of ERCs that are implemented by the target contract. The ERCs listed MUST be fully implemented. This list MAY be empty.
-    - `auditHash` - MUST be the hash of the original audit. This allows to verify that this onchain information belongs to a specific audit.
-    - `auditUri` - SHOULD point to a source where the audit can be retrieved
+    - `auditor`: Information on the auditor
+    - `auditedContract`: MUST be the `chainId` as well as `implementation` of the deployed contract the audit is related to
+    - `issuedAt`: MUST contain the information when the original audit (identified by the `auditHash`) was issued
+    - `ercs`: A list of ERCs that are implemented by the target contract. The ERCs listed MUST be fully implemented. This list MAY be empty
+    - `auditHash`: MUST be the hash of the original audit. This allows onchain verification of information that may belong to a specific audit
+    - `auditUri`: SHOULD point to a source where the audit can be retrieved
+- Contract
+    - `chainId`: MUST be a `bytes32` representation of the [ERC-155](./eip-155.md) chain ID of the blockchain that the contract has been deployed in
+    - `implementation`: MUST be a `bytes32` representation of a contract's implementation as described below
+
+### Contract Implementation
+
+Security audits generally cover a specific snapshot of an overall codebase, meaning that consequent updates of the code are not to be considered covered by it. While smart contracts are immutable on a blockchain, multiple EIP standards have been defined (i.e. ERC, ERC, etc.) that permit a "circumvention" of this blockchain trait and allow the logic of a smart contract to be updated. 
```

Comment:
Should we add specific ERC numbers here?

Reply:
Oops, yep you are right I kind of missed this one! Will add the minimal proxy standard as well as the Diamond proposition.

Reply:
Done